xcode/Subconscious/Shared/Services/ComboGeist.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ComboGeist.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 24/9/2022.
+//
+
+import Foundation
+
+struct ComboGeist: Geist {
+    private let database: DatabaseService
+
+    init(database: DatabaseService) {
+        self.database = database
+    }
+
+    func ask(query: String) -> Story? {
+        guard let stubA = database.readRandomEntry() else {
+            return nil
+        }
+        guard let stubB = database.readRandomEntry() else {
+            return nil
+        }
+        return Story.combo(StoryCombo(entryA: stubA, entryB: stubB))
+    }
+}
+
+extension ComboGeist {
+    init(
+        database: DatabaseService,
+        data: Data
```

Comment:
FYI, this form exists to allow us to easily load a JSON Tracery grammar from the app package. If we don't want to include Tracery-generated prompts with the two transcludes, then we don't need it.

However, I would lean toward including a small (possibly Tracery-generated) prompt with the story, just to frame up what you're supposed to do with these two juxtaposed stories.

Possible prompts:

- How might these be connected?
- What similarities can I see between these? Differences? 
- Can I combine these to create something new?
- Can I find three ways in which these might be connected?
- ...other good questions?

Reply:
Ah, that makes sense! It did feel like the structure was evaporating once I removed the grammar dependency. I agree that a small prompt is a nice touch, I'll re-introduce tracery and wire that up.